packages/ui/src/machines/authenticator/signUp.ts

```diff
@@ -281,16 +281,22 @@ export function createSignUpMachine({ services }: SignUpMachineOptions) {
           const { formValues, loginMechanisms } = context;
           const [primaryAlias] = loginMechanisms ?? ['username'];
 
+          let phoneNumberWithCountryCode;
           if (formValues.phone_number) {
-            formValues.phone_number =
```

Comment:
> The fix ensures that formValues.phone_number is not mutated on each signUp call.

You said it! Mutations are a huge risk!

I'm redoing #911, but that fixes `phone_number` to *always be a computed value in the form* so that this logic doesn't need to live in the machine.